src/platyPS/platyPS.psm1

```diff
@@ -304,6 +304,85 @@ function Get-PlatyPSTextHelpFromMaml
     }
 }
 
+function New-PlatyPSCab
+{
+    [Cmdletbinding()]
+    param(
+        [parameter(Mandatory=$true)]
+        [string] $Source,
+        [parameter(Mandatory=$true)]
+        [string] $Destination,
+        [parameter(Mandatory=$true)]
+        [string] $Module,
+        [parameter(Mandatory=$true)]
+        [string] $Locale,
```

Comment:
Maybe add VerifyScript to it?
I.e. I'm always not sure, do I need to say "en-US" or something else for the default one.